webapp/src/main/webapp/themes/wilma/css/wilma.css

```diff
@@ -495,37 +599,50 @@ ul#header-nav a.log-out {
     padding-left: 10px;
 }
 /* SEARCH ------>  */
+
 #search {
+    width: 100%;
+    margin-top: 25px;
     position: absolute;
-    width: 396px;
+    max-width: 393px;
     height: 38px;
     top: 40px;
-    right: 10px;
-    padding-top: 5px;
+    right: 0;
+    padding: 5px 10px 0px;
 }
+
+@media (min-width: 768px) {
```

Comment:
Please merge all media queries to be together, i.e. line 552-567 and L615-618 should be merged